package.json

```diff
@@ -4,9 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "node src/backend/index.js"
```

Comment:
Since we do not have a testing suite set up, I think we should leave this as the original text.